libs/wire-api/src/Wire/API/Event/FeatureConfig.hs

```diff
@@ -72,7 +71,6 @@ taggedEventDataSchema =
       TeamFeatureDigitalSignatures -> tag _EdFeatureWithoutConfigChanged (unnamed schema)
       TeamFeatureAppLock -> tag _EdFeatureApplockChanged (unnamed schema)
       TeamFeatureFileSharing -> tag _EdFeatureWithoutConfigChanged (unnamed schema)
-      TeamFeatureClassifiedDomains -> tag _EdFeatureClassifiedDomainsChanged (unnamed schema)
       TeamFeatureConferenceCalling -> tag _EdFeatureWithoutConfigChanged (unnamed schema)
       TeamFeatureSelfDeletingMessages -> tag _EdFeatureSelfDeletingMessagesChanged (unnamed schema)
```

Comment:
My linter says that the pattern match is not exhaustive and `TeamFeatureClassifiedDomains` is not matched. Not sure if this is an issue or not?

Reply:
This actually causes the build to fail.